translator/translate/metrics/metrics.go

```diff
@@ -53,6 +53,8 @@ func (m *Metrics) ApplyRule(input interface{}) (returnKey string, returnVal inte
 					outputPlugInfo = translator.MergeTwoUniqueMaps(outputPlugInfo, val.(map[string]interface{}))
 				} else if key == "metric_decoration" {
 					addDecorations(key, val, outputPlugInfo)
+				} else if key == "drop_origin" {
+					addDropOriginals(key, val, outputPlugInfo)
```

Comment:
I will try combine the two cases together and have it handled by one same helper method, but I don't see a way to avoid a switch(if/else) statement here when iterate thought all the keys.